### PR TITLE
Small batch of visual updates

### PR DESCRIPTION
**Pull request by MadsThy**
## Description

Small batch of visual and functional updates before demo

### CODE REVIEW

File: libs/eo/dashboard/shell/src/lib/eo-dashboard-shell.component.ts [hunk 1]
```diff
@@ -31,18 +31,16 @@ import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
...
-
       <p>
         The Energy Origin Platform is <strong>under development</strong> and new
         functionalities will be released continuously. The first release of the
         platform only offers <strong>data for companies</strong>. Data for
-        private users is intended to form part of one of the next releases. More
-        functionality will be released on an ongoing basis. If you want to
-        influence the new functionality, join us at our
+        private users is intended to form part of one of the next releases. If
+        you want to influence the new functionality, join us at our
```
Comment (Exobitt) [inline libs/eo/dashboard/shell/src/lib/eo-dashboard-shell.component.ts:39]: Grammar. Sounds weird.

**Review by robertrosborg — APPROVED** [review]

File: libs/eo/landing-page/shell/src/lib/eo-landing-page-notification.component.ts [hunk 2]
```diff
@@ -39,12 +39,18 @@ import { WattIconModule, WattIconSize } from '@energinet-datahub/watt';
...
-
       <p>
         The Energy Origin Platform is <strong>under development</strong> and new
-        functionalities will be released continuously. For now there is
-        <strong>only this page</strong>, soon it will be possible for companies
-        to login, to see the first bit of functionality.
+        functionalities will be released continuously. The first release of the
+        platform only offers <strong>data for companies</strong>. Data for
+        private users is intended to form part of one of the next releases. If
+        you want to influence the new functionality, join us at our
```
Comment (Exobitt) [inline libs/eo/landing-page/shell/src/lib/eo-landing-page-notification.component.ts:47]: Same as before

File: libs/eo/consumption-page/shell/src/lib/eo-consumption-page-info.component.ts [hunk 3]
```diff
@@ -57,6 +58,12 @@ export class EoConsumptionPageInfoComponent {
   totalMeasurement$ = this.store.totalMeasurement$;
 
   constructor(private store: EoConsumptionStore) {}
+
+  convertTokWh(wattHour: number): number {
```
Comment (Exobitt) [inline libs/eo/consumption-page/shell/src/lib/eo-consumption-page-info.component.ts:62]: kWh should be KWh

File: libs/eo/consumption-page/shell/src/lib/eo-consumption-chart-card.component.ts [hunk 4]
```diff
@@ -30,7 +31,7 @@ import { EoConsumptionStore } from './eo-consumption.store';
       <div *ngIf="(loadingDone$ | async) === false" class="loadingObfuscator">
         <watt-spinner [diameter]="100"></watt-spinner>
       </div>
-      <eo-line-chart [data]="(measurements$ | async) || []"></eo-line-chart>
+      <eo-line-chart [data]="getDataInkWh()"></eo-line-chart>
```
Comment (Exobitt) [inline libs/eo/consumption-page/shell/src/lib/eo-consumption-chart-card.component.ts:34]: kWh should be KWh

**Review by Exobitt — CHANGES_REQUESTED** [review]

**Pull request merged**